components/select_team/index.ts

```diff
@@ -12,13 +12,14 @@ import {Permissions} from 'mattermost-redux/constants';
 import {haveISystemPermission} from 'mattermost-redux/selectors/entities/roles';
 import {getSortedListableTeams, getTeamMemberships} from 'mattermost-redux/selectors/entities/teams';
 import {getCurrentUser} from 'mattermost-redux/selectors/entities/users';
+import {GlobalState} from 'mattermost-redux/types/store';
```

Comment:
This should actually be imported from `types/store` in the web app. That type extends the `mattermost-redux` one to add some app-specific state.

Reply:
Thanks. Made this change.